db/migrate/20200723214611_add_unique_indexes_to_security_events.rb

```diff
@@ -0,0 +1,10 @@
+class AddUniqueIndexesToSecurityEvents < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  def change
+    add_index :security_events, ["jti", "user_id", "issuer"], name: "index_security_events_on_jti_and_user_id_and_issuer", unique: true, algorithm: :concurrently
```

Comment:
Should the name here match that in `db/schema.rb`?

```suggestion
    add_index :security_events, ["jti", "user_id", "issuer"], name: "index_security_events_on_user_id_jti_and_issuer", unique: true, algorithm: :concurrently
```

(or vice versa)

Reply:
whooops yes, great catch thanks. I'll re-create schema.db

Reply:
fixed in 14e0615